lib/Doctrine/Query.php

```diff
@@ -921,7 +921,7 @@ public function processPendingSubqueries()
 
             $subquery = $this->createSubquery();
 
-            $sql = $subquery->parseDqlQuery($dql, false)->getQuery();
+            $sql = $subquery->parseDqlQuery($dql, false)->getSqlQuery();
```

Comment:
This is supposed to be a string (which is what `getSqlQuery` returns), and `getQuery` doesn't exist in this situation, probably a type-o.